doctr/utils/data.py

```diff
@@ -64,13 +65,17 @@ def download_from_url(
 
     Returns:
         the location of the downloaded file
+
+    Note:
+        You can change cache directory location by using `DOCTR_CACHE_DIR` environment variable.
     """
 
     if not isinstance(file_name, str):
         file_name = url.rpartition('/')[-1]
 
     if not isinstance(cache_dir, str):
-        cache_dir = os.path.join(os.path.expanduser('~'), '.cache', 'doctr')
+        cache_dir = os.environ.get(DOCTR_CACHE_DIR_ENV_NAME, os.path.join("~", ".cache", "doctr"))
```

Comment:
```
cache_dir = os.path.expanduser(
    os.environ.get(DOCTR_CACHE_DIR, os.path.join("~", ".cache", "doctr"))
)
```
or 
```
cache_dir = os.path.expanduser(DOCTR_CACHE_DIR_PATH)
```

Reply:
I would prefer the second one :)

Reply:
BTW: there is also an use in: https://github.com/mindee/doctr/blob/26ed13221757a4f13df62bcfff86e868c1b1dbe3/doctr/datasets/datasets/base.py#L99
we should update this also if the env var is set

Reply:
```
cache_dir = os.environ.get(
    "DOCTR_CACHE_DIR", 
    os.path.join(os.path.expanduser("~"), ".cache", "doctr")
) if cache_dir is None else cache_dir
```